python/swagger_client/models/alert.py

```diff
@@ -373,8 +373,8 @@ def resolve_query(self, resolve_query):
         :param resolve_query: The resolve_query of this Alert.  # noqa: E501
         :type: Query
         """
-        # if resolve_query is None:
-        #     raise ValueError("Invalid value for `resolve_query`, must not be `None`")  # noqa: E501
+        if resolve_query is None:
```

Comment:
I believe it was manually commented out before.
We should track those changes and make sure we apply them every time